plugins/com.gwtplugins.gwt.eclipse.core/plugin.xml

```diff
@@ -706,6 +706,11 @@
             class="com.google.gwt.eclipse.core.properties.ui.GWTProjectPropertyPage"
             id="com.gwtplugins.gwt.eclipse.core.properties.ui.gwtProjectPropertyPage"
             name="General Settings">
+         <enabledWhen>
```

Comment:
Hmm… `GWTProjectPropertyPage` assumes [an `IJavaProject`](https://github.com/gwt-plugins/gwt-eclipse-plugin/blob/master/plugins/com.gwtplugins.gwt.eclipse.core/src/com/google/gwt/eclipse/core/properties/ui/GWTProjectPropertyPage.java#L247).

Ah the page is a subpage of `com.gwtplugins.gdt.eclipse.core.properties.coreProjectPropertyPage` (defined in `.gdt.eclipse.core`) which is conditioned on the `com.gwtplugins.gdt.eclipse.core.isGpeSelected` expression, [defined as](https://github.com/gwt-plugins/gwt-eclipse-plugin/blob/master/plugins/com.gwtplugins.gdt.eclipse.core/plugin.xml#L12):
```
        <definition id="com.gwtplugins.gdt.eclipse.core.isGpeSelected">
            <and>
               <adapt type="org.eclipse.core.resources.IProject">
                   <test property="org.eclipse.core.resources.projectNature"
                         value="org.eclipse.jdt.core.javanature">
                   </test>
               </adapt>
            </and>
        </definition>
```
So it doesn't appear on non-Java projects anyways.